cryptofeed/feed.py

```diff
@@ -52,6 +54,7 @@ def __init__(self, address, pairs=None, channels=None, config=None, callbacks=No
                           VOLUME: Callback(None),
                           FUNDING: Callback(None),
                           OPEN_INTEREST: Callback(None),
+                          BOOK_DELTA: Callback(None),
```

Comment:
Delta callbacks aren't called by the feed implementation itself. `BOOK_DELTA` should't be called unless a callback is actually passed to the feed. So a default should not be necessary.

Reply:
is it not called from inside `cryptofeed.feed.Feed#book_callback`  ?

or is that part only every reached if `self.do_deltas = True`

Reply:
you don't need to put book deltas in here, you can remove this line